### ISSUE TITLE
Pls allow us to adjust the width of Copilot (wider)

### ISSUE THREAD

**Issue by cogsci2** [kiteco/issue-tracker#468]
On a 4k monitor, it's only a small sliver and hard to read.  I can  increase the font size but then it's even "narrower".

**Comment by tblakex01** [comment]
Agreed on a 4k monitor it's difficult to read. Adding a method to increase font size would be extremely helpful to those of us with hidpi monitors. 

Thanks

**Comment by cogsci2** [comment]
This month, I'm trying to work without Kite but you can increase font size by hitting `ctrl +`.  But copilot's width limit makes that nearly unusable.  I don't see any reason why there should be an enforced width limit at all.
